manimlib/scene/scene_file_writer.py

```diff
@@ -299,10 +306,14 @@ def combine_movie_files(self):
             '-f', 'concat',
             '-safe', '0',
             '-i', file_list,
-            '-c', 'copy',
+            #'-c', 'copy',
```

Comment:
You should do this conditionally to avoid breaking changes.

Reply:
Apparently ffmpeg can't can't convert to .gif with this option enabled, that's why I removed it.

Reply:
That's fine, but you should only remove it if the `-i` flag was passed.

Reply:
Sure, I'll add that as soon as possible